src/tools/rbd/action/ImageMeta.cc

```diff
@@ -233,6 +233,13 @@ int execute_set(const po::variables_map &vm) {
     return -EINVAL;
   }
 
+  if ((key.compare("rbd_cache") == 0) || (key.compare("rbd_cache_writethrough_until_flush") == 0)) {
+    if ((value.compare("true") != 0) && (key.compare("false") != 0)) {
```

Comment:
if ((value.compare("true") != 0) && (**# key**.compare("false") != 0)) {  

the 'key' must be value?